compose/README.md

```diff
@@ -13,7 +13,13 @@ This Docker Compose template provisions a Kong container with a Postgres databas
 To run this template execute:
 
 ```shell
-$ docker-compose up
+$ make kong-postgres
+```
+
+or run kong DB less(Alert: Kong API readonly):
+
+```shell
+$ make kong-dbless
```

Comment:
Our makefile is a developer helper tool, but we prefer that docker remains the main focus of our readme. Please modify these changes so that docker is used first and foremost. It is ok to mention the makefile as a helper for programmers running things locally afterwards.